File: 백준/Bronze/5597. 과제 안 내신 분．．？/과제 안 내신 분．．？.js
const students = new Array(30).fill(0);

for (let i = 0; i < 28; ++i) {
    const input = parseInt(require('fs').readFileSync('/dev/stdin').toString().trim().split('\n')[i]);
    students[input - 1] = input;
}

for (let j = 0; j < students.length; ++j)
    if (students[j] === 0)
        console.log(j + 1);

File: 백준/Bronze/5597. 과제 안 내신 분．．？/과제 안 내신 분．．？.cs
using System;

class Program {
    static void Main(string[] args) {
        int[] students = new int[30];
        int input = 0;
        
        for (int i = 0; i < 28; ++i) {
            input = int.Parse(Console.ReadLine());
            students[input - 1] = input;
        }
        
        for (int j = 0; j < students.Length; ++j) {
            if (students[j] == 0)
                Console.WriteLine(j + 1);
        }
    }
}